.github/prompts/generate-vividus-story-api.prompt.md

```
@@ -0,0 +1,220 @@
+---
+agent: 'agent'
+description: 'Generate VIVIDUS API test automation stories from OpenAPI/Swagger specifications. Creates executable .story files for API endpoints following VIVIDUS syntax and project conventions.'
+argument-hint: 'Provide OpenAPI specification URL or file path...'
+---
+
+## Process Overview
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)